index.html

```diff
@@ -1,523 +1,1896 @@
 <!DOCTYPE html>
 <html lang="zxx">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=yes"
+    />
+    <link rel="stylesheet" href="style.css" />
 
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=yes">
-        <link rel="stylesheet" href="style.css">
+    <!-- Fonts & Icons -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      defer
+      src="https://use.fontawesome.com/releases/v5.13.0/js/all.js"
+      integrity="sha384-ujbKXb9V3HdK7jcWL6kHL1c+2Lj4MR4Gkjl7UtwpSHg/ClpViddK9TI7yU53frPN"
+      crossorigin="anonymous"
+    ></script>
 
-        <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet">
-        <script defer src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" integrity="sha384-ujbKXb9V3HdK7jcWL6kHL1c+2Lj4MR4Gkjl7UtwpSHg/ClpViddK9TI7yU53frPN" crossorigin="anonymous"></script>
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="css/bootstrap.css" />
 
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-        <link rel="stylesheet" href="css/bootstrap.css">
+    <!--MyCustomCSS-->
+    <link rel="stylesheet" href="css/custom.css" />
 
-        <link rel="stylesheet" href="css/custom.css">
+    <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+  </head>
 
-        <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
-    </head>
+  <body class="m-0 customBody">
+    <header class="clearfix">
+      <div
+        class="p-3 b-0 d-flex flex-wrap align-items-center justify-content-between w-100 customHeader"
+      >
+        <div
+          class="float-left w-25 d-none d-lg-block d-none d-lg-block text-nowrap text-left m-0"
+        >
+          <p>
+            <img
+              class="w-25 p-2"
+              src="https://g.newsweek.com/img/weather/26.png"
```

Comment:
You also did a good job here, and we are almost there. 

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88579212-12bb0a00-d020-11ea-9c69-37a8c777fe8c.png) | ![image](https://user-images.githubusercontent.com/61479648/88579286-31b99c00-d020-11ea-8bd0-77799693c614.png) |

- [ ] It should have less height

- [ ] The hamburger Icon should not show up in the large screen sizes

- [ optional ] Try to make this as the original
:arrow_down: 

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/88579428-64639480-d020-11ea-85c7-097fc68f6869.png) | ![image](https://user-images.githubusercontent.com/61479648/88579535-8d842500-d020-11ea-8839-31a04f02ca3c.png) |